src/Main.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Integer> numbersList = new ArrayList<>();
+
+        for (String arg : args) {
+            numbersList.add(Integer.parseInt(arg));
+        }
+
+        System.out.println(numbersList.size());
+        System.out.println(numbersList);
+
+        squaredNumbers(numbersList);
+
+        printAllNumbersAndSumNumbers(numbersList);
+    }
+
+    private static void squaredNumbers(List<Integer> list) {
+        List<Integer> squared = list.stream()
+                .map(x -> x * x)
+                .collect(Collectors.toList());
+        System.out.println(squared);
+    }
+
+    private static void printAllNumbersAndSumNumbers(List<Integer> list) {
```

Comment:
I would change method name to something shorter, like `printNumberSum`

Reply:
@wojciechkostecki when you resolved issue from a comment mark conversation as resolved with button below: "Resolve conversation"